src/installation/guides/chroot.md

```diff
@@ -0,0 +1,313 @@
+
+Unpack the tarball into the newly configured filesystems.
+
+```
+# tar xvf void-[...]-ROOTFS.tar.xz -C /mnt
+```
+
+## Configuration
+
+> Note: With the exception of the section "Install base-system (ROOTFS method
+> only) the remainder of this guide is common to both XBPS and ROOTFS
+> installation methods.
+
+### Entering the Chroot
+
+Mount the pseudo-filesystems needed for a chroot,
+
+```
+# mount --rbind /sys /mnt/sys && mount --make-rslave /mnt/sys
+# mount --rbind /dev /mnt/dev && mount --make-rslave /mnt/dev
+# mount --rbind /proc /mnt/proc && mount --make-rslave /mnt/proc
+```
+
+Copy the DNS configuration into the new root so that XBPS can still download new
+packages inside the chroot,
+
+```
+# cp /etc/resolv.conf /mnt/etc/
+```
+
+Chroot into the new installation
+
+```
+# PS1='(chroot) # ' chroot /mnt/ /bin/bash
+```
+
+### Install base-system (ROOTFS method only)
+
+> Note: This section can be skipped if following the XBPS method. You already
+> have base-system installed.
+
+ROOTFS images generally contain out of date software, due to being a snapshot of
+the time when they were built, and do not come with a complete `base-system`.
+Update the package manager and install `base-system`.
+
+```
+# xbps-install -Su xbps
+# xbps-install -u
+# xbps-install base-system
+# xbps-remove base-voidstrap
+```
+
+### Installation Configuration
+
+Write the hostname to `/etc/hostname`. Go through the options in
+[`/etc/rc.conf`](../../config/rc-files.md#rcconf). If installing a glibc
+distribution, edit `/etc/default/libc-locales`, uncommenting desired
+[locales](../../config/locales.md).
+
+[nvi(1)](https://man.voidlinux.org/nvi.1) is available in the chroot, but you
+may wish to install your preferred text editor at this time.
+
+For glibc builds, generate locale files with
+
+```
+(chroot) # xbps-reconfigure -f glibc-locales
+```
+
+### Set a Root Password
+
+[Configure at least one super user account.](../../config/users-and-groups.md)
+Other user accounts can be configured later, but there should either be a root
+password, or a new user account with [sudo(8)](https://man.voidlinux.org/sudo.8)
+privileges.
+
+To set a root password, run
+
+```
+(chroot) # passwd
+```
+
+### Configure fstab
+
+Configure [fstab(5)](https://man.voidlinux.org/fstab.5). This file can be
+automatically generated from currently mounted filesystems by copying the file
+`/proc/mounts`.
+
+```
+(chroot) # cp /proc/mounts /etc/fstab
+```
+
+Remove lines in `/etc/fstab` that refer to `proc`, `sys`, `devtmpfs` and `pts`.
+
+Replace references to `/dev/sdXX`, `/dev/nvmeXnYpZ`, etc. with their respective
+UUID, which can be found by found by running
+[blkid](https://man.voidlinux.org/blkid.8). Referring to filesystems by their
+UUID guarantees they will be found even if they are assigned a different name at
+a later time. In some situations, such as booting from USB, this is absolutely
+essential. In other situations, such as SATA or NVME, drives will usually always
```

Comment:
"usually", or "always", but not both. :-)

Reply:
I absolutely definitely agree with that. I'm not sure yet what to put in its place so I'm going to leave it for the time being.

The sense I want to get across is that if you're using sata, it's likely (usually) that everytime the computer boots (always), the drives will have the same name. I'm not sure that either word on their own quite captures that. I'll take another look at this later today and make a change.

Reply:
Okay, I've settled on saying that in some situations drives will always have the same name, but took out the example scenario it might happen in. I don't think there's anything wrong with having /dev/sda1 in your fstab, but I think it's bad general advice. I don't want to suggest to new users that they should just skip that step because it might break unexpectedly down the line.